styles/languages/gfm.less

```diff
@@ -1,12 +1,11 @@
 .source.gfm {
   .markup {
     -webkit-font-smoothing: auto;
-    &.heading {
-      color: @hue-5;
-    }
+  }
 
-    &.link {
-      color: @hue-3;
+  .list {
+    .variable {
+      color: @hue-5;
```

Comment:
This seems not even needed, since those "list variables" already gets styled as stand alone `.variable`.

I think it can be removed.